.env.template

```diff
@@ -1,3 +1,4 @@
 DEPLOYER_PRIVATE_KEY=
 PROPOSER_PRIVATE_KEY=
 MAINNET_RPC_URL=
+FOUNDRY_FUZZ_SEED=
```

Comment:
I'd suggest removing this, and specifying the seed in the `lite` profile of the config:
- This is easier as users don't need another env var,
- This is also the intent of the `lite` profile: settings to quickly run tests for faster iteration

Reply:
Done!